.flake8

```diff
@@ -1,12 +1,14 @@
 [flake8]
-exclude=
-    .tox
-    setuptools/_vendor,
```

Comment:
Removing the trailing comma seems wrong: the official docs show an example with the commas present — https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-exclude.

Reply:
I did some testing and it seems like commas are indeed unnecessary.